### PR TITLE
Adds Maven Central link.

### PR DESCRIPTION
**Pull request by coollog**

### CODE REVIEW

File: minikube-maven-plugin/README.md [hunk 1]
```diff
@@ -4,7 +4,7 @@ Minikube Maven Plugin
 ======================
 This plugin provides tasks and configuration to manage a minikube lifecycle for Maven developers.
 
-Now available on [Maven Central]().
+Now available on [Maven Central](https://mvnrepository.com/artifact/com.google.cloud.tools/minikube-maven-plugin).
```
Comment (elharo) [inline minikube-maven-plugin/README.md]: This is not maven central.

Comment (coollog) [reply minikube-maven-plugin/README.md]: Changed the wording to just say `the Central repository`.

File: minikube-maven-plugin/README.md [hunk 2]
```diff
@@ -4,7 +4,7 @@ Minikube Maven Plugin
 ======================
 This plugin provides tasks and configuration to manage a minikube lifecycle for Maven developers.
 
-Now available on [Maven Central](https://mvnrepository.com/artifact/com.google.cloud.tools/minikube-maven-plugin).
+Now available on the [Central repository](https://mvnrepository.com/artifact/com.google.cloud.tools/minikube-maven-plugin).
```
Comment (elharo) [inline minikube-maven-plugin/README.md]: The problem is the URL, not the wording. mvnrepository.com is *NOT* Maven Central. You want maven.org

Comment (coollog) [reply minikube-maven-plugin/README.md]: Okay, changed the link to point to the latest Maven Central link (same as the badge).

**Pull request merged**